Project-Euler/Project-Euler_UnitTests/Problems_UnitTests.cs

```diff
@@ -0,0 +1,75 @@
+﻿using NUnit.Framework;
+using Project_Euler;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Tests
+{
+    [TestFixture]
+    public class Problems_UnitTests
+    {
+        [Test]
+        [TestCase("233168")]
```

Comment:
This is technically correct, but generally you only use "TestCase" if you have multiple scenarios to tests.

So for all these problems since there is only 1 thing to test, I would remove the test case and just put the expected string in the assert.